modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/worker/CuratedPodcastsCrawler.kt

```diff
@@ -15,15 +15,15 @@ import kotlinx.coroutines.coroutineScope
 import timber.log.Timber
 
 class CuratedPodcastsCrawler(
-    private val service: ListWebService,
+    private val listRepository: ListRepository,
     private val staticHostUrl: String,
 ) {
     @Inject constructor(
-        service: ListWebService,
-    ) : this(service, BuildConfig.SERVER_STATIC_URL)
+        listRepository: ListRepository,
+    ) : this(listRepository, BuildConfig.SERVER_STATIC_URL)
 
-    suspend fun crawl(platform: String): Result<List<CuratedPodcast>> = coroutineScope {
-        runCatching { service.getDiscoverFeed(platform) }.mapCatching { discover ->
+    suspend fun crawl(): Result<List<CuratedPodcast>> = coroutineScope {
+        runCatching { listRepository.getDiscoverFeed() }.mapCatching { discover ->
```

Comment:
The discover feed now takes a version number so I switched it over to using the list repository instead of accessing the Retrofit class directly.